### DISCUSSION TITLE
Is it possible to use dark and light custom icons?

### DISCUSSION THREAD

**Discussion by jorge-campo** [squidfunk/mkdocs-material#4229] [Q&A] [answered]
I see the built-in Material for MkDocs icons change their color automatically when changing the theme: white icons under the dark theme and black icons in the light theme. This happens without adding any extra code to the Markdown file.

Is it possible to achieve the same effect with custom icons?

I may be wrong, but I haven't seen this question posted before and can't find any information in the Material for MkDocs documentation.

Thanks!

**Comment by squidfunk** [comment, marked as answer]
Yes that's definitely possible. You must make sure that your SVG does not contain any [`fill`](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill) or [`stroke`](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke) attributes. This is likely added by vector graphic programs when exporting SVGs, so remove them and you should see the colors taking effect.

**Reply by jorge-campo** [reply]
Thanks, @squidfunk ! I see the SVG icons in Figma do not contain these attributes. The question is, what should I do now? Should I drop the dark and light versions of these icons in the `.icons` folder, and Material will pick up the correct version?

**Reply by squidfunk** [reply]
In order to use the native color switching functionality, there needs to be only one icon version without `fill` attributes. The theme will then fill in the colors:

https://github.com/squidfunk/mkdocs-material/blob/82f3b1a34754b2abcfc178edb0b9ce447da27255/src/assets/stylesheets/main/extensions/pymdownx/_emoji.scss#L37-L41

**Reply by squidfunk** [reply]
Also, see [our logo](https://github.com/squidfunk/mkdocs-material/tree/master/src/.icons) for reference.

**Reply by jorge-campo** [reply]
Thank you! I tested this, and it works 🙂
